generate-offline-map/README.md

```diff
@@ -34,9 +34,8 @@ The map used in this sample shows the [stormwater network](https://arcgisruntime
 
```

Comment:
On line `17` could you update step 1 to use: "`PoralItem`" instead of "`Portal` item"